react-ui/web/src/websocket/misc_ws.js

```diff
@@ -25,4 +46,14 @@ function receiveServerTime (callback) {
     });
 }
 
-export { connectToWebsocket, getServerTime, receiveServerTime };
\ No newline at end of file
+function receiveMonitoringShiftData (callback) {
+    connectToWebsocket();
+    socket.on("receive_monitoring_shifts", data => {
+        const temp = JSON.parse(data);
+        console.log("SHIFTS: ", temp);
+        callback(temp);
+    });
+    
+}
+
+export { subscribeToMiscWebSocket, connectToWebsocket, unsubscribeToMiscWebSocket, getServerTime, receiveServerTime, receiveMonitoringShiftData };
```

Comment:
```suggestion
export {
    subscribeToMiscWebSocket, connectToWebsocket,
    unsubscribeToMiscWebSocket, getServerTime,
    receiveServerTime, receiveMonitoringShiftData
};

```

needed new line at end of file